src/app/modules/item/pages/item-forum/item-forum.component.ts

```diff
@@ -132,8 +153,30 @@ export class ItemForumComponent implements OnInit, OnChanges, OnDestroy {
     this.refresh$.next();
   }
 
+  toggleThread({
+    rowData,
+    itemId,
+    participantId
+  }:{
+      rowData: Thread & { isOpened: boolean, rowIndex: number },
+      itemId: string,
+      participantId: string,
+    }
+  ): void{
+    this.discussionService.toggleVisibility(rowData.isOpened === false, {
```

Comment:
[⚠️ personal opinion]
Please, for boolean `rowData.isOpened === false` -> `!rowData.isOpened` and `isStuff === true` -> `isStuff`. A boolean is already an expression, otherwise you end up with `(isStuff === true) === true`. I was removing point to my students for that years ago 😁